presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -95,6 +121,18 @@ public ClientSession toClientSession()
                 debug);
     }
 
+    public KerberosConfig toKerberosConfig()
+    {
+        KerberosConfig config = new KerberosConfig()
+                .setConfig(new File(krb5ConfigPath))
+                .setKeytab(new File(krb5KeytabPath));
+        if (krb5CredentialCachePath == null) {
+            krb5CredentialCachePath = System.getenv("KRB5CCNAME");
```

Comment:
addressed. but since Krb5LoginModule.java has already got the logic to parse various types of credential paths, maybe it's better to change the type of KerberosConfig's credentialCache from File to String.